artemis/src/main/java/tech/pegasys/artemis/BeaconNode.java

```diff
@@ -118,10 +117,10 @@ public void stop() {
 final class EventBusExceptionHandler
     implements SubscriberExceptionHandler, ChannelExceptionHandler {
 
-  private final StatusLogger logger;
+  private final Logger log;
```

Comment:
These are the kinds of messages we almost certainly want going to STDOUT when in "interactive" mode rather than being hidden in a log file.  The spec failure suggests a block or attestation failed to process and the unexpected error definitely needs to be brought to the users attention.

Is there a plan for how to display this kind of output which isn't the routine status updates, but should still go to the "UI"?

Reply:
Thanks for the additional context 👍 

> Is there a plan for how to display this kind of output which isn't the routine status updates, but should still go to the "UI"?
Not yet, the need for that extra bucket was only slowly dawning on me. It suggests the `StatusLogger` may need splitting into two classes (one handling status updates and the other this kind UI feedback) 🤔